### ISSUE TITLE
input budgets don't update after opening pointers

### ISSUE THREAD

**Issue by LiamDonovan4** [oughtinc/mosaic#716]
Sometimes updating input the budget can take minutes to hours (i.e. only updates when I get assigned to the workspace a second time) after I open a locked pointer, making it difficult to keep track of input budget. One workspace even jumped from ~+65 to ~-35 (!) characters the next time I was assigned it, even though no new pointers had been opened in the meantime (I don't have the link). 

Here's a [link ](https://mosaic.ought.org/w/7467?e=efcded60-7f33-41f3-9d88-8e1ae677e017) to the latest workspace where this has happened to me. It could easily be some internet problem on my end, but it would be nice if the input budget updated even when the internet connection drops.

**Comment by zjmiller** [comment]
Wasn't able to replicate this issue, but updating the displayed input budget does require an internet connection, so we could restructure the UI to optimistically update here.

**Comment by LiamDonovan4** [comment]
Yes, that would explain it (I was connected to very spotty wi-fi at the time)
